ccmlib/scylla_docker_cluster.py

```diff
@@ -48,18 +50,21 @@ def _update_config(self):
 class ScyllaDockerNode(ScyllaNode):
     def __init__(self, *args, **kwargs):
         kwargs['save'] = False
+        # This line should appear before the "supper" method
+        self.share_directories = ['data', 'commitlogs', 'conf', 'hints', 'view_hints', 'saved_caches', 'keys']
         super(ScyllaDockerNode, self).__init__(*args, **kwargs)
         self.base_data_path = '/usr/lib/scylla'
         self.local_base_data_path = os.path.join(self.get_path(), 'data')
         self.local_yaml_path = os.path.join(self.get_path(), 'conf')
-        self.docker_name = f'{self.cluster.get_path().split("/")[-1]}-{self.cluster.name}-{self.name}'
+        folder_name = self.cluster.get_path().split("/")[-2]
+        self.docker_name = f'{folder_name}-{self.cluster.name}-{self.name}'
         self.jmx_port = "7199"  # The old CCM code expected to get a string and not int
         self.log_thread = None
 
     def _get_directories(self):
         dirs = {}
-        for i in ['data', 'commitlogs', 'conf', 'logs', 'hints', 'view_hints', 'saved_caches']:
-            dirs[i] = os.path.join(self.get_path(), i)
+        for dir_name in self.share_directories + ['logs', 'hints']:
```

Comment:
i see `hints` is already part of `self.share_directories`
in this case, shouldn't `logs` be part of that too?

Reply:
Fixed, thanks for notice